src/scripts/main.js

```diff
@@ -1,3 +1,67 @@
 'use strict';
 
 // write code here
+const rowAppend = document.querySelector('.append-row');
+const rowRemove = document.querySelector('.remove-row');
+const columnAppend = document.querySelector('.append-column');
+const columnRemove = document.querySelector('.remove-column');
+const table = document.querySelector('table');
+const max = 9;
+const min = 3;
```

Comment:
```suggestion
const MAX = 9;
const MIN = 3;
```
константы записывают большими буквами обычно